xbmc/interfaces/json-rpc/JSONServiceDescription.cpp

```diff
@@ -1424,7 +1424,7 @@ bool CJSONServiceDescription::prepareDescription(std::string &description, CVari
   }
 
   if (description.at(0) != '{')
-    description = StringUtils::Format("{{{:s}}}", description);
+    description = StringUtils::Format("{{{:s}}}", description.c_str());
```

Comment:
Is this commet wrong then?
https://github.com/xbmc/xbmc/blob/master/xbmc/utils/StringUtils.h#L52